Uchu.World/Objects/Components/ReplicaComponents/CharacterComponent.cs

```diff
@@ -429,6 +429,12 @@ public long Currency
             }
         }
 
+        public long CurrencySilent
+        {
+            get => _currency;
+            set => _currency = value;
+        }
+
```

Comment:
Not sure I like the design of `CurrencySilent` and `UniverseScoreSilent`. To me, these seem like they should be method calls since should only really be a setter.

Reply:
Maybe. I did it like this so you can write `character.CurrencySilent += 100` just as you can write `character.Currency += 100`, it would look inconsistent if one was `character.AddCurrencySilently(100)` or something like that, while now you can just type Silent and it'll do it silently.

Reply:
Replaced them with methods to increment the score/currency silently